frontend/src/reducer/workspaceContentList.js

```diff
@@ -29,53 +30,84 @@ export const serializeContent = c => ({
   created: c.created
 })
 
-export default function workspaceContentList (state = [], action) {
+export default function workspaceContentList (state = {}, action) {
```

Comment:
add a defaultState constant to initialize the state with the properties workpaeId and contentList